library/ssl_tls13_keys.c

```diff
@@ -133,6 +136,60 @@ static void ssl_tls13_hkdf_encode_label(
     *dst_len = total_hkdf_lbl_len;
 }
 
+#if defined( MBEDTLS_TEST_HOOKS )
```

Comment:
I do not understand this pre-compiler directive. I would say that we want to define and compile `mbedtls_psa_hkdf_extract` even if MBEDTLS_TEST_HOOKS is not defined, right ?

Reply:
ok fine then.